azure-mgmt-monitor/src/main/java/com/microsoft/azure/management/monitor/MetricAlert.java

```diff
@@ -154,6 +176,26 @@
              * @return the next stage of metric alert definition.
              */
             WithWindowSize withTargetResource(HasId resource);
+
+            /**
+             * Sets specified resources as target to alert on metric. All resources must be of same type and in same region.
+             *
+             * @param resourceIds collection of resource id to alert on metric.
+             * @param type resource type.
+             * @param regionName resource region.
+             * @return the next stage of metric alert definition.
+             */
+            @Beta(Beta.SinceVersion.V1_25_0)
```

Comment:
Beta is not right. Should we update the base lib for it ?

Reply:
> Beta is not right. Should we update the base lib for it ?

Do you know where it is? I didn't find the annotation repository that time.